src/getCoinCombination.test.js

```diff
@@ -7,4 +7,39 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
```

Comment:
Please, add tests that will check some hardcoded values, like 69, and expect it to equal some combination. Because you do not have any complicated examples here.